SAKPaaS/src/app/components/search-bar/search-bar.component.ts

```diff
@@ -65,4 +65,8 @@ export class SearchBarComponent implements OnInit {
       this.searchService.triggerSearch(this.searchControl.value);
     }
   }
+
+  clearSearch(): void {
+    this.searchControl.setValue('');
```

Comment:
Aufbauend auf dem Kommentar von  @patrickstecker um die Locations nach dem clearen wieder zu laden. Was vielleicht auch (optional) noch schön wäre, wäre wenn man nach dem klicken auf das kreuz den Eingabemodus verlässt :)

Reply:
Ich hab das mit dem clearen jetzt noch hinzugefügt.
Nur das mit dem Eingabemodus verlassen hab ich keine Ahnung, wie ich das in einer PWA umsetzen soll tbh, das könnte man als künftiges Feature vielleicht mit aufnehmen